cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -483,6 +486,52 @@ def test_delete_static_page(self, store):
         resp = self.client.delete(reverse_usage_url('xblock_handler', usage_key))
         self.assertEqual(resp.status_code, 204)
 
+    @patch('openedx.core.djangoapps.bookmarks.api.delete_bookmarks')
```

Comment:
Once you've switched to a signals-based pattern, please move these tests out of contentstore and into the bookmarks app.